include/fs.h

```diff
@@ -414,6 +414,21 @@ int fs_mount(struct fs_mount_t *mp);
  */
 int fs_unmount(struct fs_mount_t *mp);
 
+/**
+ * @brief Mount point read entry
+ *
+ * Reads mount point entries
+ *
+ * @param number Pointer to mount point number
+ * @param name Pointer to mount point name
+ *
+ * @retval 0 Success
+ * @retval -ERRNO errno code if error
+ * @return In end-of-mount condition, this will return 0 and set
```

Comment:
what is end-of-mount condition?

Reply:
Correct this would be more clear, I will update the code. 

I will also add a comment to indicate that the parameter `number` will be incremented if a valid mount point is returned.

Reply:
ENOEXIST does not exist in zephyrs errno.h file using ENOENT instead.